packages/amplication-build-manager-service/src/codeBuild/codeBuild.service.ts

```diff
@@ -11,49 +11,115 @@ import {
 } from '@aws-sdk/client-codebuild';
 import { Injectable } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
-import { CODE_BUILD_PROJECT_NAME } from 'src/constants';
+import { join } from 'path';
+import {
+  BUILD_ARTIFACT_S3_BUCKET,
+  BUILD_ARTIFACT_S3_LOCATION,
+  BUILD_IMAGE_NAME,
+  BUILD_IMAGE_VERSION,
+  CODE_BUILD_PROJECT_NAME,
+  GET_BUILD_BY_RUN_ID_TOPIC,
+} from 'src/constants';
```

Comment:
Remove `src/` imports everywhere please. I have already refactored the entire codebase to get rid of them.

Reply:
Done